app/src/main/java/com/a5corp/weather/SplashScreen/SplashScreen.java

```diff
@@ -0,0 +1,46 @@
+package com.a5corp.weather.SplashScreen;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.a5corp.weather.GlobalActivity;
+import com.a5corp.weather.R;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SplashScreen extends AppCompatActivity {
+    private TextView tv;
+    private ImageView iv;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash_screen);
+        tv = findViewById(R.id.tv);
+        iv = findViewById(R.id.iv);
+        Animation myanim = AnimationUtils.loadAnimation(this , R.anim.mytransition);
+        tv.startAnimation(myanim);
+        iv.startAnimation(myanim);
+        final Intent i = new Intent(this, GlobalActivity.class);
```

Comment:
Oh no, I will have to reject this PR. Simply doing a `sleep()` will only increase the time taken to display the weather data.

As I had said in #27 , `GlobalActivity.java` must be the splash screen. In this you check whether the city has been set or not. If not, it goes to Enter City screen. If it has been set, `GlobalActivity.java` will load the weather data in this splash screen and send the data received to the WeatherActivity, where the WeatherFragment should parse it.